main.py

```diff
@@ -336,7 +336,8 @@ def read_client_config(name, client_config):
         client_config.get("languageId", ""),
         client_config.get("enabled", True),
         client_config.get("initializationOptions", dict()),
-        client_config.get("settings", dict())
+        client_config.get("settings", dict()),
+        client_config.get("environment", dict())
```

Comment:
Let's call it `env`, just like the build system (and as I'd requested in the issue)